sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -565,6 +565,33 @@ class ExplainSuiteAE extends ExplainSuiteHelper with EnableAdaptiveExecutionSuit
          |""".stripMargin
     )
   }
+
+  test("SPARK-35133: explain codegen should work with AQE") {
+    withSQLConf(SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> "true") {
```

Comment:
Hi, @c21 . 
Although SPARK-33679 enabled AQE by default for Apache Spark 3.2, we don't know what happens during QA and Apache Spark 3.2.0 RC period. In addition, we had better an explicit configuration in the test case. Could you add `spark.sql.adaptive.enabled=true` explicitly?

Reply:
@dongjoon-hyun - sorry if it's not clear, this test is under `ExplainSuiteAE` which extends `EnableAdaptiveExecutionSuite` to enforce AQE enabled already.

Reply:
Oh, got it. Then, it's perfect.